x12-edi/2.11/modules/ROOT/pages/x12-edi-connector-reference.adoc

```diff
@@ -0,0 +1,406 @@
+| Segment terminator a| String |  Segment terminator character. The configured value is used by default for all output messages, but can be overridden at the message level. |  ~ |
+| Segment line ending a| Enumeration, one of:
+
+** NONE
+** LF
+** CRLF
+** CR |  Line ending to add between segments to improve the readability of the output message text. |  NONE |
+| Unique group numbers a| Boolean a|  Whether to send globally unique Group Control Numbers (GS06):
+
+* If `true`, the connector assigns unique group numbers across all interchanges sent to the same partner and application (as determined by the interchange sender and receiver identification, combined with the functional group sender and receiver application codes).
+* If `false`, the connector assigns functional group control numbers sequentially within each interchange and reuses the numbers in different interchanges.  |  false |
+| Unique transaction numbers a| Boolean a|  Whether to send globally unique Transaction Set Control Numbers (ST02): 
+
+* If `true`, the connector assigns unique transaction set numbers across all interchanges sent to the same partner and application (as determined by the interchange sender and receiver identification, combined with the functional group sender and receiver application codes).
+* If `false`, the connector assigns transaction set control numbers sequentially within each functional group and reuses the control numbers in different groups. 
+ |  false |
+| Implementation reference a| String |  Implementation convention reference for transactions (ST03). If this value is set, it is used as the ST03 Implementation Convention Reference unless it is overridden in the message parameters. |  |
+| Initial interchange number a| Number |  Initial Interchange Control Number used for outgoing messages. |  1 |
+| Initial group number a| Number |  Initial Group Control Number used for outgoing messages. |  1 |
+| Initial transaction number a| Number |  Initial Transaction Set Control Number used for outgoing messages. |  1 |
+| Requests acknowledgments a| Boolean a|  Whether to request acknowledgments such as TA1 for the sent transactions flag (ISA14):
+
+* If `true`, TA1 acknowledgments are requested for all sent transactions.
+* If `false`, TA1 acknowledgements are not requested.|  false |
+| Default usage indicator a| String a|  Default interchange usage indicator (ISA15): 
+
+* `I`: Information
+* `P`: Production data
+* `T`: Test data |  P |
+|Set acknowledgement schema by inbound message a| Boolean a| If the *acknowledgmentSchemaPath* is `EMPTY` and this property is `true` it will use the *Version / Release / Industry Identifier Code (GS08)* field value from the inbound message to set the Functional/Implementation Acknowledgment structure. When `false` it uses a hardcoded Functional/Implementation Acknowledgment structure of the 005010 version. | false |
```

Comment:
```suggestion
|Set acknowledgement schema by inbound message a| Boolean a| If the *Acknowledgement schema path* field is `EMPTY` and this property is `true`, it will use the *Version / Release / Industry Identifier Code (GS08)* field value from the inbound message to set the Functional/Implementation Acknowledgment structure. When `false`, it uses a hardcoded Functional/Implementation Acknowledgment structure of the 005010 version. | false |
```